DQM/DTMonitorClient/src/DTDAQInfo.cc

```diff
@@ -79,7 +80,8 @@ void DTDAQInfo::dqmEndLuminosityBlock(DQMStore::IBooker& ibooker,
 
     daqMap->Reset();
     //get fed summary information
-    sumFED = &setup.getData(sumFEDToken_);
+    ESHandle<RunInfo> sumFED;
```

Comment:
I am not sure how to make the migration on this since RunInfoRcd has not getData function
Any advice?

Reply:
I finally managed to